src/openassetio-core/tests/utils/RegexTest.cpp

```diff
@@ -1,9 +1,12 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright 2023 The Foundry Visionmongers Ltd
-#include <fmt/format.h>
+// Copyright 2025 The Foundry Visionmongers Ltd
```

Comment:
Surely 2023-2025 for consistency with other copyright updates?

Reply:
I'm seeing a number of updates like this, just changing the year, while others have a range. What is the update policy?

Reply:
Yeah this is wrong.  I set up my IDE (CLion) to auto-update Copyright years, but it's hit and miss. Sometimes it doesn't update it (e.g. if the change was due to an automatic refactor), and when it does update it, it doesn't introduce a range when it should (like in this case you highlight).  

I am aware of it, and need to do a pass over all modified files and fix the headers.  Another artifact of the fact that I PR'd this accidentally and too early.

Reply:
All Copyright headers of modified files should now be correct, i.e. IIRC they're all a range like `Copyright YYYY-2025` (don't think there's any file newly created in 2025).